src/status_im/chat/handlers/input.cljs

```diff
@@ -334,11 +334,11 @@
                                 (dispatch [:update-seq-arguments chat-id])
                                 (dispatch [:send-current-message]))]
         (dispatch [::request-command-data
-                   {:content   command
-                    :chat-id   chat-id
-                    :jail-id   (or (get-in command [:command :bot]) chat-id)
-                    :data-type :validator
-                    :after     #(dispatch [::proceed-validation %2 after-validation])}])))))
+                   {:content     command
+                    :chat-id     chat-id
+                    :jail-id     (or (get-in command [:command :bot]) chat-id)
+                    :data-type   :validator
+                    :event-after #(vector ::proceed-validation %2 after-validation)}])))))
```

Comment:
sometimes we use first argument sometimes second, maybe it's better to use `fn[name1 name2]` to better understand arguments?